app/javascript/angular/code/filtersUtils.js

```diff
@@ -59,23 +59,15 @@ export const setupTimeRangeFilter = (callback, timeFrom, timeTo) => {
 
 export const setupAutocomplete = (callback, id, path) => {
   if (document.getElementById(id)) {
-    $("#" + id)
-      .bind("keydown", function(event) {
```

Comment:
This was the core of the problem

Reply:
cool, so the library handles that on it's own

Reply:
It seems so